detection_rules/devtools.py

```diff
@@ -918,7 +918,7 @@ def trim_version_lock(min_version: str, dry_run: bool):
             for outdated in outdated_vers:
                 popped = lock['previous'].pop(str(outdated))
                 if outdated == latest_version:
-                    lock['previous'][str(min_version)] = popped
+                    lock['previous'][Version.__str__(min_version[:2])] = popped
```

Comment:
The initial CLI command was developed and merged 06/02/2022 which was tested then, but the schema for version lock file allowed a full semantic version. This then changed 06/07/2022 where the schema only supported major and minor semantic versions, but the CLI command was not tested and thus up to this point needed to be adjusted.

[CLI command added](https://github.com/elastic/detection-rules/pull/2013/files#diff-8dcef89fe672ab07924dae360eeb04a04b69fb7d66c60f702359920769399827)
[Schema updated](https://github.com/elastic/detection-rules/pull/2021/files#diff-1b7f9f4101b33ab4640395a3ac7fe8c8ba7d9a3d6f9a4844f9b19458792897a5)